bin/resources/GameIndex.yaml

```diff
@@ -9930,7 +9930,7 @@ SCPS-55003:
 SCPS-55004:
   name: "ジャック×ダクスター 旧世界の遺産"
   name-sort: "じゃっくんだくすたー きゅうせかいのいさん"
-  name-en: "Jak x Daxter - Kyuu Sekai no Isan"
+  name-en: "Jak and Daxter - Kyuu Sekai no Isan"
```

Comment:
this  was already correct

Reply:
Does that mean that the one written in Jak and Daxter was incorrect in terms of English?

Does that mean we should go with Jak x Daxter?

Reply:
==> No change